e2e-test/src/interfaces.rs

```diff
@@ -42,6 +42,17 @@ pub static INTERFACE_NAMES: &[&str] = &[
     SERVER_PROPERTY_NAME,
 ];
 
+pub static INTERFACE_NAMES_WITH_ADDED: &[&str] = &[
+    ADDITIONAL_DEVICE_DATASTREAM_NAME,
+    ADDITIONAL_SERVER_DATASTREAM_NAME,
+    DEVICE_AGGREGATE_NAME,
+    DEVICE_DATASTREAM_NAME,
+    DEVICE_PROPERTY_NAME,
+    SERVER_AGGREGATE_NAME,
+    SERVER_DATASTREAM_NAME,
+    SERVER_PROPERTY_NAME,
+];
```

Comment:
Can't this be just the array of the additional interfaces? Then we can join both the arrays during the e2e test.